File: src/main/java/ar/edu/um/tesoreria/rest/model/Documento.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class Documento extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -4488497020106093903L;

	@Id
	@Column(name = "doc_id")
	private Integer documentoId;

	@Column(name = "doc_nombre")
	private String nombre = "";

}

File: src/main/java/um/tesoreria/rest/kotlin/model/Documento.kt
package um.tesoreria.rest.kotlin.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class Documento(

    @Id
    @Column(name = "doc_id")
    var documentoId: Int? = null,

    @Column(name = "doc_nombre")
    var nombre: String = "",

    ) : Auditable()
